src/components/HomePage/HomePage.css

```diff
@@ -0,0 +1,6 @@
+#home-background {
+  background: url('../../static/img/home_background.jpg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  height: calc(100vh - 120px);
```

Comment:
@MartyTheeMartian Keep the aspect ratio of the background and don't stretch it; show its native size even if the screen is large, so it's not pixelated.